cpp/include/cudf/io/parquet_metadata.hpp

```diff
@@ -173,15 +196,18 @@ class parquet_metadata {
    * @param num_rows number of rows
    * @param num_rowgroups number of row groups
    * @param file_metadata key-value metadata in the file footer
+   * @param rg_metadata vector of maps containing metadata for each row group
```

Comment:
Generally we avoid abbreviations.
```suggestion
   * @param rowgroup_metadata vector of maps containing metadata for each row group
```

Reply:
I guess it's fine then.